### PR TITLE
Newformat

### PR DESCRIPTION
**Pull request by zhfnjust**

### CODE REVIEW

File: contracts/asm.scrypt [hunk 1]
```diff
@@ -45,8 +45,8 @@ contract Asm {
     }
 
     function lenFail(bytes b) : int {
-        // this is wrong since the last value on stack will be considered as the return value
         asm {
+            // this is wrong since the last value on stack will be considered as the return value
```
Comment (zhfnjust) [inline contracts/asm.scrypt:49]: need more ast info

File: contracts/matrix.scrypt [hunk 2]
```diff
@@ -21,15 +18,9 @@ contract Matrix {
...
-                          [3, 3, 3, 3],
-                          [4, 4, 4, 4]];
-        
-        int[N][N] mat1 = [[1, 1, 1, 1],
-                          [2, 2, 2, 2],
-                          [3, 3, 3, 3],
-                          [4, 4, 4, 4]];
+        int[N][N] mat0 = [[1, 1, 1, 1], [2, 2, 2, 2], [3, 3, 3, 3], [4, 4, 4, 4]];
+
+        int[N][N] mat1 = [[1, 1, 1, 1], [2, 2, 2, 2], [3, 3, 3, 3], [4, 4, 4, 4]];
```
Comment (zhfnjust) [inline contracts/matrix.scrypt:23]: still TODO: no break line now

File: contracts/modExp.scrypt [hunk 3]
```diff
@@ -2,7 +2,7 @@ contract ModExp {
     // max # of bits for e = ceil(log2(y))
     static const int N = 232;
     // modulus
-    const int M;
+    int M;
```
Comment (zhfnjust) [inline contracts/modExp.scrypt:5]: need fix

**Review by xhliu — APPROVED** [review]
Move auction.js into a separate commit. It's not related.

**Pull request merged**